src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -374,13 +383,115 @@ impl DragonballInner {
         Ok((old_vcpus, new_vcpus))
     }
 
+    pub(crate) fn resize_memory(
+        &mut self,
+        old_mem_mb: u32,
+        new_mem_mb: u32,
+    ) -> Result<(u32, MemoryConfig)> {
+        // check the invalid request memory
+        if new_mem_mb > self.hypervisor_config().memory_info.default_maxmemory
+            && self.hypervisor_config().memory_info.default_maxmemory > 0
```

Comment:
default_maxmemory's type is u32, is always greater than 0, so checking here is useless.